frontend/src/lib/v2/CompareUtils.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MetricsType } from 'src/pages/CompareV2';
+
+export const metricsTypeToString = (metricsType: MetricsType): string => {
```

Comment:
Can we just put the corresponding string into the enum definition here?
https://github.com/kubeflow/pipelines/blob/master/frontend/src/pages/CompareV2.tsx#L118-L124
```
enum MetricsType {
   SCALAR_METRICS = 'Scalar Metrics',
   CONFUSION_MATRIX = 'Confusion Matrix',
   ...
}
``` 
Then, you don't have to create this `CompareUtils.ts` file.

Reply:
@zijianjoy James, what do you think?

Reply:
No problem at all, thanks Joe!